### PR TITLE
Add IPFS Desktop

### PR DESCRIPTION
**Pull request by mikabak**
Requested here #623

### CODE REVIEW

**Review by philclifford — COMMENTED** [review]
LGTM

File: 01-main/packages/ipfs-desktop [hunk 1]
```diff
@@ -0,0 +1,11 @@
+DEFVER=1
+ARCHS_SUPPORTED="amd64"
```
Comment (natanjunges) [inline 01-main/packages/ipfs-desktop:2]: This is already the default, so not needed.

**Review by natanjunges — CHANGES_REQUESTED** [review]

File: 01-main/packages/ipfs-desktop [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+get_github_releases "https://api.github.com/repos/ipfs/ipfs-desktop/releases/latest"
```
Comment (philclifford) [inline 01-main/packages/ipfs-desktop:3]: Recent merges mean this should be changed to
```suggestion
get_github_releases "ipfs/ipfs-desktop" "latest"
```

File: 01-main/packages/ipfs-desktop [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+if [ "${ACTION}" != "prettylist" ]; then
+    URL="$(grep "browser_download_url.*\.deb\"" "${CACHE_DIR}/${APP}.json" | head -n1 | cut -d'"' -f4)"
```
Comment (philclifford) [inline 01-main/packages/ipfs-desktop:5]: and also this one
```suggestion
    URL="$(grep "browser_download_url.*\.deb\"" "${CACHE_FILE}" | head -n1 | cut -d'"' -f4)"
```

File: 01-main/packages/ipfs-desktop (hunk 1 above)
Comment (philclifford) [inline 01-main/packages/ipfs-desktop:2]:
```suggestion
```

**Review by philclifford — CHANGES_REQUESTED** [review]
Please make the suggested updates to align with recent merges in `main`

**Review by natanjunges — APPROVED** [review]

**Pull request merged**